recipe/recipe.yaml

```
@@ -2,18 +2,18 @@ schema_version: 1
 
 context:
   name: azure-batch
-  version: 14.2.0
+  version: 15.0.0b2
 
 package:
   name: ${{ name|lower }}
   version: ${{ version }}
 
 source:
-  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/azure-batch-${{ version }}.tar.gz
-  sha256: c79267d6c3d3fe14a16a422ab5bbfabcbd68ed0b58b6bbcdfa0c8345c4c78532
+  url: https://files.pythonhosted.org/packages/30/f5/8eb6c79cfc5df4081fdd11cc7c731ab0a7cfb196a6af07a83453b5b3ce43/azure_batch-${{ version }}.tar.gz
```

Don't use this path, keep the old pattern.

---

@xhochy, that seems to fail:

```
│ ╭─ Fetching source code
 │ │ Fetching source from url: https://pypi.org/packages/source/a/azure-batch/azure-batch-15.0.0b2.tar.gz
 │ │
 │ ╰─────────────────── (took 0 seconds)
 │
 ╰─────────────────── (took 0 seconds)
Error:   × Failed to download source from url: HTTP status client error (404 Not
  │ Found) for url (https://files.pythonhosted.org/packages/source/a/azure-
  │ batch/azure-batch-15.0.0b2.tar.gz)
  ╰─▶ HTTP status client error (404 Not Found) for url ([https://](https://github.com/conda-forge/azure-batch-feedstock/pull/9)
      files.pythonhosted.org/packages/source/a/azure-batch/azure-batch-
      15.0.0b2.tar.gz)
```

Are pre-released stored in the same index as regular releases on PyPI?

---

@janjagusch, I think you just need to change the dash to an underscore:
```suggestion
  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/azure_batch-${{ version }}.tar.gz
  sha256: 624ca3811a8205b3deab243998e139409bebf7f1ef4abe9e76cf3b1a7d1038fd
```

---

For some reason, some packages change at times on PyPI from having dashes to having underscores.

---

I' m ready to approve as soon as we get the URL fixed.